include/rmlab/renderer/lines2svg.cpp

```diff
@@ -0,0 +1,339 @@
+/* Copyright 2017-2018 Axel Huebl
```

Comment:
@matteodelabre pls feel free to add your name to this file :)
(Is it ok if in ASCII? I have weird experiences with some compilers with non-ascii comments in source files)

Reply:
Haha, thanks. Same with my last name ;)